docs/source/keywords.rst

```diff
@@ -160,6 +160,10 @@ General options
       * Aggressively vectorize loops
       * ``OFF``
 
+    * * ``Kokkos_ENABLE_COMPILE_AS_CMAKE_LANGUAGE``
+      * Build with the CMake language feature.
```

Comment:
It is not very clear, can you list affected languages?

Reply:
I listed CUDA and HIP